control/latlon_muxer/CMakeLists.txt

```diff
@@ -1,33 +1,42 @@
-cmake_minimum_required(VERSION 3.0.2)
+cmake_minimum_required(VERSION 3.5)
 project(latlon_muxer)
 
-add_compile_options(-std=c++14)
+### Compile options
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  # add_compile_options(-Wall -Wextra -Wpedantic)
```

Comment:
Question: Do we want to include these compile options in all packages or exclude them? For the package I've been working on, I've included these.

Reply:
I'd use them in all packages, I forgot to uncomment this one while I was fixing the lint errors. Will push an updated version. Thanks for catching that.

Reply:
In general, I would suggest to then add `-Werror` as well. Just for porting, we can keep the warnings as they are, but for safer code in the future, we should not allow warnings caused by our code

Reply:
@fred-apex-ai done in 17229ba